config/secrets.yml

```diff
@@ -1,8 +1,13 @@
 development:
   secret_mail_base: <%= ENV["SECRET_MAIL_BASE"] %>
+  mailer_domain: <%= ENV["MAILER_DOMAIN"] %>
+  mailer_password: <%= ENV["MAILER_PASSWORD"] %>
+  mailer_user_name: <%= ENV["MAILER_USER_NAME"] %>
+  mailer_address: <%= ENV["MAILER_ADDRESS"] %>
+  mailer_port: <%= ENV["MAILER_PORT"] %>
 
 test:
  secret_mail_base: <%= ENV["SECRET_MAIL_BASE"] %>
 
 production:
-  secret_mail_base: <%= ENV["SECRET_MAIL_BASE"] %>
\ No newline at end of file
+  secret_mail_base: <%= ENV["SECRET_MAIL_BASE"] %>
```

Comment:
You're using ENV variables in the config for the different ones but didn't add them to the secrets